packages/static-wado-webserver/lib/routes/study.mjs

```diff
@@ -0,0 +1,11 @@
+import serverSCPFallbackController from "../controllers/serverSCPController.mjs";
+
+/**
+ * Set studies (/studies) routes.
+ *
+ * @param {*} router root entry point for studies routes.
+ */
+export default function setRoutes(router) {
```

Comment:
This needs to get the params that this route is getting set with, so you need to include the configuration object.